infrastructure/cdn-in-a-box/optional/grafana/trafficcontrol-scenes/.eslintrc

```diff
@@ -0,0 +1,39 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/*
+ * ⚠️⚠️⚠️ THIS FILE WAS SCAFFOLDED BY `@grafana/create-plugin`. DO NOT EDIT THIS FILE DIRECTLY. ⚠️⚠️⚠️
+ *
+ * In order to extend the configuration follow the steps in
+ * https://grafana.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/extend-configurations#extend-the-eslint-config
+ */
+{
+  "extends": ["@grafana/eslint-config"],
```

Comment:
We really need a master ruleset at the root that each project extends from. Right now I just keep asking people to try to stick to the rules in the TPv2 project directory, which is understandably confusing and annoying. You don't have to do that now (at least partially because you'd have to know what subset of those rules to use as the base) but ideally the rules here would line up at least a little with the others - e.g. indentation and quote style, since they're the most pervasive.

Reply:
Understood. I copied the rules from TPv2 and only change a bit